atomicapp/install.py

```diff
@@ -72,7 +72,8 @@ def _copyFromContainer(self, image):
                           ''.join(random.sample(string.letters, 6)))
         logger.debug("Creating a container with name %s", name)
 
-        create = [self.docker_cli, "create", "--name", name, image, "nop"]
+        # Workaround docker bug BZ1252168 by using run instead of create
+        create = [self.docker_cli, "run", "--name", name, image, "nop"]
```

Comment:
This seems broken. The 'nop' cmd is not valid with `run`.

```
$ sudo docker run --name projectatomic/atomicapp nop
usage: atomicapp [-h] [-V] [-v] [-q] [--dry-run] [-a ANSWERS]
                 {run,install,stop} ...
atomicapp: error: argument action: invalid choice: 'nop' (choose from 'run', 'install', 'stop')
```

I think you need to override using entrypoint.

```
$ sudo docker run --entrypoint /bin/true projectatomic/atomicapp
$ echo $?
0
```